src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -45,95 +39,103 @@
       - sudo
       - systemd
       - tcpdump
-      - tig
-      - tmate
-      - tmux
-      - vim
```

Comment:
We want these on every container, not just the devel one. Please, avoid functional changes.

Reply:
Well, we do not want these on anything else, they are not needed nor wanted for test environment. 
None of these is needed for testing and some of them are missing from rhel baseos/appstream

Reply:
Actually gdb, tcpdump and wireshark also do not make sense on all machines.
They got a specific usecases mostly on client.

Reply:
Moved to extended package set.